lib/bankin/resources/category.rb

```diff
@@ -0,0 +1,19 @@
+module Bankin
+  class Category < Resource
+    RESOURCE_PATH = '/v2/categories'
+
+    has_fields :id, :name
+
+    has_resource :parent, 'Category'
+
+    def self.list(options = {})
```

Comment:
**.list** and **.get** methods are accessible for authorized bankin users and should accept **token** argument like **Account** resource. Also, it should be described in **User** resource as _auth_delegable_
```
auth_delegate :categories, class: 'Category', method: :list
auth_delegate :category, class: 'Category', method: :get
```
In this case you'll be able to call Bankin::Category methods on related resources like:
```ruby
bankin_user = Bankin::User.authenticate('email', 'password')

bankin_user.categories
bankin_user.category(id)
bankin_user.transactions.first.category
```

Reply:
Done in 60f4ee9.